about-me.md

```diff
@@ -2,7 +2,7 @@
 
 ![Picture of me]()
 
-Hi, I'm ________. I'm a ________ at *__________*.
+Hi, I'm Charlene Bernard. I'm a ________ at *__________*.
```

Comment:
i thought your name is sheri??? WHO ARE YOU?

Reply:
My legal name is Charlene. But only my mom calls me that, when I'm in trouble! Also on government forms. 🤷‍♀

Reply:
Ah makes sense. Cool cool!